### ISSUE TITLE
'csvddfVersion' was unexpected

### ISSUE THREAD

**Issue by rgaiacs** [frictionlessdata/frictionless-py#1176]
Consider the minimal working example:

```
{
  "profile": "tabular-data-resource",
  "name": "iris",
  "path": "iris.csv",
  "dialect": {
    "csvddfVersion": 1.2,
    "delimiter": ",",
    "header": true,
    "commentChar": "#"
  }
}
```

When I use `frictionless` to validate it, I get

```
$ frictionless --version
frictionless --version
4.40.3
$ frictionless validate tabular-data-resource/mwe.json

# -------
# invalid: iris.csv 
# -------

## Summary 

+-------------------------------+-------------------+
| Description                   | Size/Name/Count   |
+===============================+===================+
| File name (Not Found)         | iris.csv          |
+-------------------------------+-------------------+
| File size                     | N/A               |
+-------------------------------+-------------------+
| Total Time Taken (sec)        | 0.003             |
+-------------------------------+-------------------+
| Total Errors                  | 1                 |
+-------------------------------+-------------------+
| Dialect Error (dialect-error) | 1                 |
+-------------------------------+-------------------+

## Errors 

+-------+---------+----------+--------------------------------------------------+
| row   | field   | code     | message                                          |
+=======+=========+==========+==================================================+
|       |         | dialect- | Dialect is not valid: "Additional properties are |
|       |         | error    | not allowed ('csvddfVersion' was unexpected)" at |
|       |         |          | "" in metadata and at "additionalProperties" in  |
|       |         |          | profile                                          |
+-------+---------+----------+--------------------------------------------------+
```

Based on https://specs.frictionlessdata.io/csv-dialect/, `csvddfVersion` is a valid property.

**Comment by shashigharti** [comment]
Thank you for reporting it! We will investigate it.

**Comment by roll** [comment]
Hi I've released a fix in v4.40.4
